app/assets/stylesheets/application.scss

```diff
@@ -6,3 +6,4 @@
 @import "bento.scss";
 @import "aleph.scss";
 @import "feedback.scss";
+@import "results.scss";
```

Comment:
is there a reason you broke this out into it's own partial? I'd prefer the code in it go with the other result styling, so I can move it all here (if the compile is allowing variables) or I can move the new code into bento. Let me know what you prefer, or if you want to fight about it.